npm/private/npm_translate_lock_generate.bzl

```diff
@@ -22,7 +22,7 @@ _NPM_IMPORT_TMPL = \
 """
 
 _BIN_TMPL = \
-    """load("{repo_package_json_bzl}", _bin = "bin", _bin_factory = "bin_factory")
+    """load("@{repo_package_json_bzl}", _bin = "bin", _bin_factory = "bin_factory")
```

Comment:
hmm. this should probably be fixed below where the `repo_package_json_bzl` template param is expanded and the additional `@` added depending of bzlmod is supported